apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/entity/po/Permission.java

```diff
@@ -30,7 +30,7 @@
  */
 @Entity
 @Table(name = "Permission")
-@SQLDelete(sql = "Update Permission set isDeleted = 1 where id = ?")
+@SQLDelete(sql = "Update Permission set IsDeleted = 1, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000) where Id = ?")
```

Comment:
Does it need to let `DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000)` to be a constant?

```java
public class SQLConstants {
  public static final String SOFT_DELETE_UPDATE 
      = "IsDeleted = 1, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000)";
}
```

then reference this constant

```suggestion
@SQLDelete(sql = "Update Permission set " + SQLConstants.SOFT_DELETE_ UPDATE + " where Id = ?")
```

Reply:
Although the function `ROUND(UNIX_TIMESTAMP(NOW(4))*1000)` is a little bit complex, I prefer to keep the columns on the SQL cause it will be more clear.

@nobodyiam How do you think?

Reply:
I think this level of duplication is ok since it's clear in the SQL clauses.